tools/codesigningtool/codesigningtool.py

```diff
@@ -0,0 +1,84 @@
+import argparse
```

Comment:
in general for this script, could you make the method names more descriptive, mark them all as private with the _ prefix, and add a docstring explaining what it does?

Reply:
👍